### PR TITLE
[#363] Implemented Playback Screen

### LINKED ISSUE #363: Meditation Playback Screen UX
**Actual Behaviour.**  
Missing Meditation Playback Screen UX.

**Expected Behaviour**  
Meditation Playback Screen UX. 

**Steps to reproduce it**  
Click on a Meditation to meditate according to guide.

**Additional context**  
> Not available 

**Would you like to work on the issue?**
Yes.

### PR DESCRIPTION
**Pull request by jddeep**
Fixes #363 

**Changes**: Implemented Playback Screen

**Screenshot/s for the changes**: 

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [ ] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: 
https://drive.google.com/open?id=10unqEESv8gnComCOE1o6zcU68aCIg-PH

**References**:
I used a track from with different names for now:
* https://freemusicarchive.org/about

### CODE REVIEW

**Comment by CloudyPadmal** [comment]
Align the play pause rewind icons horizontally inline. Where did you get the images? Are they open source?

**Comment by jddeep** [comment]
The icons are vector assets from the default Android clip arts. The author image, i downloaded from Google images.
Will look for if any copyright issues are there on this.
Okay will align horizontally. Thought to give it a curvy touch. :sweat_smile:

**Comment by CloudyPadmal** [comment]
That's person's image. Pretty sure he didn't make it "Open" for everyone. Replace with a default image

And make all three buttons similar. Two has circular borders and one does not

**Comment by jddeep** [comment]
> That's person's image. Pretty sure he didn't make it "Open" for everyone. Replace with a default image

Haha, true. Will replace. Thanks.

**Comment by jddeep** [comment]
@CloudyPadmal The change was made. Please review and merge. Thanks.

File: app/src/main/java/io/neurolab/activities/MeditationActivity.java [hunk 1]
```diff
@@ -101,6 +89,20 @@ private void initializePlaybackController() {
         Log.d(TAG, "Inside initializePlaybackController: MediaPlayerHolder progress callback set");
     }
 
+    private String getTimeString(long millis) {
+        StringBuffer buf = new StringBuffer();
+
+        int minutes = (int) (millis % (1000 * 60 * 60)) / (1000 * 60);
+        int seconds = (int) ((millis % (1000 * 60 * 60)) % (1000 * 60)) / 1000;
+
+        buf
```
Comment (CloudyPadmal) [inline app/src/main/java/io/neurolab/activities/MeditationActivity.java]: Reformat this by removing unnecessary spacing and indents.

File: app/src/main/java/io/neurolab/activities/MeditationActivity.java [hunk 2]
```diff
@@ -149,24 +153,18 @@ public void onStateChanged(@PlaybackStateCompat.State int state) {
...
-                            }
-                        });
-            }
         }
     }
 
+    @Override
+    public void onBackPressed() {
+        super.onBackPressed();
+        startActivity(new Intent(this, MeditationListActivity.class));
```
Comment (CloudyPadmal) [inline app/src/main/java/io/neurolab/activities/MeditationActivity.java]: Why do we need to start an activity here?

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]
Are those sound tracks open and free to use? Please add reference as a comment where it's first used.

File: app/src/main/java/io/neurolab/activities/MeditationActivity.java (hunk 2 above)
Comment (jddeep) [reply app/src/main/java/io/neurolab/activities/MeditationActivity.java]: Will check and revert.

Comment (jddeep) [reply app/src/main/java/io/neurolab/activities/MeditationActivity.java]: Change made. :+1:

File: app/src/main/java/io/neurolab/activities/MeditationActivity.java (hunk 1 above)
Comment (jddeep) [reply app/src/main/java/io/neurolab/activities/MeditationActivity.java]: Change made :+1:

**Comment by jddeep** [comment]
@CloudyPadmal Please have a look now. Thanks.

**Comment by CloudyPadmal** [comment]
@jddeep 
`© 2019 New Mindful Life. All Rights Reserved. `
`© 2019 Penguin Random House`

Copyrighted content is serious stuff. I think we've to write to them and get permission to use these. Remove the resource for now.

CC: @mariobehling

**Comment by jddeep** [comment]
@CloudyPadmal I see. Alright, will remove those resources for now. Maybe for now we can use some dummy tracks. What do you suggest?

**Comment by jddeep** [comment]
@CloudyPadmal 
I used one track from below source with different names for now:
* https://freemusicarchive.org/about

This can be merged now, I believe. Thanks.

**Review by CloudyPadmal — APPROVED** [review]

**Pull request merged**